pkg/schemadsl/lexer/flaggablelexer_test.go

```diff
@@ -0,0 +1,74 @@
+package lexer
+
+import (
+	"slices"
+	"testing"
+
+	"github.com/authzed/spicedb/pkg/schemadsl/input"
+)
+
+var flaggableLexerTests = []lexerTest{
+	{"use expiration", "use expiration", []Lexeme{
+		{TokenTypeIdentifier, 0, "use", ""},
+		{TokenTypeWhitespace, 0, " ", ""},
+		{TokenTypeKeyword, 0, "expiration", ""},
+		tEOF,
+	}},
+	{"use expiration and", "use expiration and", []Lexeme{
```

Comment:
What role does `and` play here?

Reply:
`relation viewer: user with caveat and expiration` - the `and` is a keyword in this case